src/loki/height_action.cc

```diff
@@ -74,17 +75,17 @@ std::vector<PointLL> loki_worker_t::init_height(valhalla_request_t& request) {
   "range_height": [ [0,303], [8467,275], [25380,198] ]
 }
 */
-std::string loki_worker_t::height(valhalla_request_t& request) {
+std::string loki_worker_t::height(Api& request) {
```

Comment:
lots of changes like t his as well where we now just pass the single pbf object around instead of the facade over it